examples/examples/balance_transfer.rs

```diff
@@ -2,20 +2,19 @@
 // This file is dual-licensed as Apache-2.0 or GPL-3.0.
 // see LICENSE for license details.
 
-//! To run this example, a local polkadot node should be running. Example verified against polkadot 0.9.18-4542a603cc-aarch64-macos.
+//! To run this example, a local polkadot node should be running. Example verified against polkadot polkadot 0.9.25-5174e9ae75b.
```

Comment:
one additional with example `OfflineClient` would be great

Reply:
What sort of thing did you have in mind? Maybe an example where we create a signed extrinsic offline and then just print out the hex for it?

Reply:
the simplest possible how to instantiate it and use it just to demonstrate how the API works... because I don't understand how it will work i.e, how would me as user use the "offline extrinsic" display QR code, send via USB/UART, etc?

create an signed extrinsic sounds good to me....

Reply:
Sounds good to me!

I think let's create an issue for this to avoid doing more work on this PR :)

Reply:
(I'll make one!)